com.github.Murmele.Gittyup.json

```diff
@@ -0,0 +1,62 @@
+{
+    "app-id": "com.github.Murmele.Gittyup",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "Gittyup",
+    "finish-args": [
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--share=network",
+        "--share=ipc",
+        "--filesystem=home",
+        /* needed to start external applications like kdiff3 (merge tool) ... */
+        "--filesystem=/usr/bin",
+        "--filesystem=/bin",
+        /*we use the keyring to store credentials*/
+        "--filesystem=xdg-run/keyring",
+        /*for git repos that require ssh keys*/
+        "--socket=ssh-auth",
+        "--talk-name=org.freedesktop.secrets",
+        "--talk-name=org.freedesktop.Notifications",
+        "--env=PATH=:/usr/bin:/app/bin:/app/bin/x86_64-linux:/app/bin/aarch64-linux"
+    ],
+    "build-options":
+     {
+        "env": {
+            "PATH": "$PATH:/usr/bin:/app/bin"
```

Comment:
this is already set this way. It's not necessary.

Reply:
thank you